src/BenchmarksApps/Samples/Template/Template.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <FrameworkReference Update="Microsoft.AspNetCore.App" RuntimeFrameworkVersion="$(MicrosoftAspNetCoreAppPackageVersion)" />
```

Comment:
Should it be conditioned in order to work locally? Or are these values already defined by the sdk?

Reply:
I am not sure. From CoreFX team member perspective I don't want to run it locally, only using the ASP.NET Perf lab

Reply:
Until someone wants to run it locally in order to debug what is not working on the server. Conditioning it is easy, I can commit the change to your PR if your are ok with that.

Reply:
> I can commit the change to your PR if your are ok with that.

I think that it would be the best option since I simply don't know what exactly to change.